packages/dashboard/src/components/eventsTable.tsx

```diff
@@ -85,7 +124,8 @@ export function EventsTable({
   const { page, pageSize } = paginationModel;
   const theme = useTheme();
   const workspace = useAppStore((store) => store.workspace);
-  const apiBase = useAppStore((store) => store.apiBase);
+  const apiBase ='https://3001-rutik3-dittofeed-b0l07s2gxv6.ws-us107.gitpod.io';
```

Comment:
Hi Rutik can you remove this? You can override this value with the environment variable `DASHBOARD_API_BASE`